index.html

```diff
@@ -290,20 +254,20 @@ <h2>Contact me</h2>
 
                     <li class="language">
                         <h4>Frameworks   </h4>
-                        <img class="arrow" src="Image/list.png" alt="arrow keys">
+                        <img class="arrow" src="Image/list.png" alt="alt-image_text"> <!-- Added alternate image text-->
```

Comment:
- [ ] The purpose of the `alt` attribute is to describe the image for people who can't see the image. Kindly use a more descriptive alternative text, and please remove the comment after the `img` tag, since this does not add functionality to your code and after some time, it will not make sense to be present in this file.

Reply:
done.